src/errors/content-not-found.error.ts

```diff
@@ -0,0 +1,14 @@
+export class ContentNotFoundError extends Error {
```

Comment:
I thought it would make sense to keep it defined at the same level as the service that use them?
But moving it up to popover utils would certainly mutualize the code. Do you think I should do it?

Reply:
Do you think the error content should be different between other frameworks (angular, react, symfony, ...)?
If yes, stay as is else it should be common in utils but perhaps in a next development to avoid to block the release...